my_calendar/lib/my_calendar/app/app.ex

```diff
@@ -0,0 +1,102 @@
+defmodule MyCalendar.App do
+  alias MyCalendar.App.{Meeting, User, MeetingsUser}
+  alias MyCalendar.Repo
+
+  import Ecto.Changeset, only: [put_assoc: 3]
+  import Ecto.Query, only: [from: 2]
+
+  @spec get_user(non_neg_integer()) :: Ecto.Schema.t() | nil
+  def get_user(id), do: Repo.get(User, id)
+
+  @spec get_meeting(non_neg_integer()) :: Ecto.Schema.t()
+  def get_meeting(id) do
+    Repo.get(Meeting, id)
+    |> Repo.preload([:organizator, :users])
+  end
+
+  @spec get_meetings_by_date(Date.t()) :: list(Ecto.Schema.t())
+  def get_meetings_by_date(date) do
+    dt_from = DateTime.new!(date, ~T[00:00:00])
+    dt_to = DateTime.new!(Date.add(date, 1), ~T[00:00:00])
+
+    Repo.all(from(m in Meeting, where: m.date_time >= ^dt_from and m.date_time < ^dt_to))
+    |> Repo.preload([:organizator, :users])
+  end
+
+  @spec get_meetings_by_organizator_id(non_neg_integer()) :: list(Ecto.Schema.t())
+  def get_meetings_by_organizator_id(organizator_id) do
+    Repo.all(from(m in Meeting, where: m.organizator_id == ^organizator_id))
+    |> Repo.preload([:organizator, :users])
+  end
+
+  @spec get_meetings_by_user_id(non_neg_integer()) :: list(Ecto.Schema.t())
+  def get_meetings_by_user_id(user_id) do
+    query =
+      from(m in Meeting,
+        join: mu in MeetingsUser,
+        on: mu.meeting_id == m.id and mu.user_id == ^user_id
+      )
+
+    Repo.all(query)
+    |> Repo.preload([:organizator, :users])
+  end
+
+  @spec list_users() :: list(Ecto.Schema.t())
+  def list_users, do: Repo.all(User)
+
+  @spec list_meetings() :: list(Ecto.Schema.t())
+  def list_meetings do
+    Repo.all(Meeting)
+    |> Repo.preload([:organizator, :users])
```

Comment:
Здесь, и много где еще: начинать пайп принято с терма.
```suggestion
    Meeting
    |> Repo.all()
    |> Repo.preload([:organizator, :users])
```